src/services/member/plugins/action/service.ts

```diff
@@ -1,15 +1,82 @@
-import { CannotModifyOtherMembers } from '../../../../utils/errors';
+import { partition } from 'lodash';
+
+import { ActionTriggers, PermissionLevel } from '@graasp/sdk';
+
+import { CannotModifyOtherMembers, UnauthorizedMember } from '../../../../utils/errors';
 import { Repositories } from '../../../../utils/repositories';
 import { ActionService } from '../../../action/services/action';
+import { validatePermissionMany } from '../../../authorization';
+import { Item, ItemExtraMap } from '../../../item/entities/Item';
 import { Actor } from '../../entities/member';
 
+export const getPreviousMonthFromNow = () => {
+  const date = new Date(); // Today's date
+  date.setMonth(date.getMonth() - 1); // Set the date to one month ago
+  return date;
+};
+
+export const actionTypesWithoutNeedOfPermission: string[] = [
+  ActionTriggers.CollectionView,
+  ActionTriggers.ItemLike,
+  ActionTriggers.ItemUnlike,
+  ActionTriggers.ItemDownload,
+  ActionTriggers.ItemEmbed,
```

Comment:
Actually all of these you need to check the permissions. At least you should be able to access it. I think with the current way you check the permissions it is okay.